server/views/case-summary-record.njk

```diff
@@ -31,26 +31,35 @@
     <div class="govuk-grid-row">
         <div class="govuk-grid-column-two-thirds">
 
+            <div class="govuk-grid-row">
+                <div class="govuk-grid-column-full">
+                    <h1 class="govuk-heading-l">Probation record</h1>
+                </div>
+            </div>
+
             {% if activeOrders | length %}
-                <h2 class="govuk-heading-m">Current orders ({{ activeOrders | length }})</h2>
+                <h2 class="govuk-heading-m">Current orders <span class="govuk-caption-l govuk-!-font-weight-bold govuk-!-display-inline-block">({{ activeOrders | length }})</span></h2>
```

Comment:
I was able to switch out the additional class and include govuk frontend styles.